pkgs/by-name/ru/runme/package.nix

```diff
@@ -13,13 +13,13 @@
 
 buildGoModule rec {
   pname = "runme";
-  version = "3.8.3";
+  version = "3.10.2";
 
   src = fetchFromGitHub {
     owner = "stateful";
     repo = "runme";
     rev = "v${version}";
-    hash = "sha256-AnGh1Kg3CV4UkEwZ81QMeIf9whdeCglmCbfp8DC6N30=";
+    hash = "sha256-KOEifQZVogw/3jPhm167YIM8PSGIwRhebDua5wRm0ZY=";
   };
 
   vendorHash = "sha256-qOM66jiSjAU5e6eGrM8kgJxWJzxwgHpLadx5pTjNFOs=";
```

Comment:
looks like this also needs to be updated

Reply:
Wouldn't it have failed with my local build?

Reply:
And CI seems fine. How do you know this needs to change? I'd like to understand

Reply:
not sure what you are looking at that makes you think CI is fine 

![image](https://github.com/user-attachments/assets/6b5d419a-0bf9-405a-aba7-93a41b8c050f)

as for why it didn't fail for your local build

https://nixos.org/manual/nixpkgs/unstable/#chap-pkgs-fetchers-caveats

Reply:
https://github.com/NixOS/nixpkgs/pull/366511/checks?check_run_id=34685521353

Reply:
I see green ticks and queued jobs, no failures. 

It seems like this could be more explicit for new contributors.

Reply:
I'll read some docs and learn how to get the Go sum hash. Thank you for the advice